### ISSUE TITLE
launch failing on bare metal

### ISSUE THREAD

**Issue by JonathanFraser** [pachyderm/pachyderm#172]
More or less a follow up to the previous issue (issue:165) I had. The kubernetes cluster comes up fine on my dev machine, but the pachyderm pod fails to deploy.

I get volume mount errors for objd, etcd and rethink containers in the pachyderm pods. I suspect this is just a general kubernetes or docker configuration error but I don't know the system well enough to extract why the volumes are failing to mount other than the ever so useful:

  FirstSeen	LastSeen	Count	From			SubobjectPath	Type		Reason		Message
  ---------	--------	-----	----			-------------	--------	------		-------
  54s		54s		1	{scheduler }						Scheduled	Successfully assigned objd-kzvho to 127.0.0.1
  54s		10s		6	{kubelet 127.0.0.1}					FailedMount	Unable to mount volumes for pod "objd-kzvho_default": operation not supported
  54s		10s		6	{kubelet 127.0.0.1}					FailedSync	Error syncing pod, skipping: operation not supported

from the kubernetes events. 

Any help would be appreciated.

**Comment by jdoliner** [comment]
Hmm, I haven't seen an error like this. Objd isn't doing anything terrible sophisticated with volumes either, it's just got a single `emptyDir` volume.

What version of Kubernetes are you using and how are you launching it? Also which storage backend are you using for Docker?

**Comment by Binaryblade** [comment]
Version of kubernetes is the one that comes up when you run launch-kube so 1.1.2
docker version: 1.9.1

as per the previous ticket the storage backend was switched to device mapper (although the underlying file system is still btrfs)

all three of those containers seem to just want an empty directory mount but I can't seem to find any errors that are more useful.

**Comment by jdoliner** [comment]
Is it possible to bring up other k8s examples or do those fail as well?

Sorry, wish I could be more useful on this but I haven't seen anything like this form k8s :(

**Comment by teodor-pripoae** [comment]
Hi,

I'm getting this error when I'm doing `make launch`. It seems on older versions kubernetes was accepting a bool pointer for privileged, but no it just takes the value. I'm running kubernetes 1.1.3

```
The ReplicationController "ppsd" is invalid.
spec.template.spec.containers[0].securityContext.privileged: forbidden '<*>(0xc20b22c4b0)true'
```

**Comment by jdoliner** [comment]
Looks like you figured this one out for yourself.
We should mention this problem somewhere on the README since others will probably run into this as well.

**Comment by JonathanFraser** [comment]
Seems like the mount errors are a issue in kubernetes, they are resolved in version 1.2. Request a version bump of the start script to kubernetes v1.2.

**Comment by jdoliner** [comment]
Request granted, 1.2 shipped six days ago. Will submit a pr.

**Comment by sjezewski** [comment]
This is merged, and we ask users to use 1.2+ specifically in the setup instructions.
